packages/mint-components/src/components/tax-and-cash/sqm-docusign-form/sqm-docusign-form.feature

```diff
@@ -0,0 +1,93 @@
+
+    Examples: 
+      | typeTaxForm | taxFormSubHeader  | taxFormDescriptionCopy                                                                                                   | changeTaxFormCopy                                                |
+      | W9          | W9 Tax Form       | Participants based in the US and partnering with US-based brands need to submit a W9 form.                               | Not based in the US?                                             |
+      | W8-BEN      | W8-BEN Tax Form   | Individuals residing outside of the US, joining the referral program of a US-based cmpany, need to submit a W8-BEN form. | Represent a business entity or you're based in the US?           |
+      | W8-BEN-E    | W8-BEN-E Tax Form | Participants residing outside of the US who represent a business enttiy need to submit a W8-BEN-E form.                  | Joining this program as an individual or you're based in the US? |
+
+  @minutia
+  Scenario: Participant successfully submits Docusign document and is directed to summary page
+    When they successfully fillout and submit the Docusign document within the iframe
+    And they press "Continue" or refresh the page
+    Then they will be directed to the Document Submitted page
+
+  @minutia @ui
+  Scenario: Docusign iframe is loading
+    When the Docusign iframe is loading the Tax Form
+    Then the iframe displays a spinnder inside the igrame
+    When the Docusign iframe loads in the Tax Form
+    Then the loader dissapers and the participant can see the Tax Form
+
+  @minutia
+  Scenario: Participant completes Docusign form
+    When they fillout and complete the Docusign form inside the iframe
+    Then the Docusign iframe session completes with one of the following success statuses:
+      | signing_complete |
+      | viewing_complete |
+    Then they are redirected to step 4
+
+  @minutia @ui
+  Scenario: Participant's Docusign session expires
+    When they have the Docusign form open in the iframe
+    And they do not interact with the iframe for more than 20 minutes
+    Then the Docusign iframe session expires with one of the following expirey statuses:
+      | session_timeout |
+      | ttl_expired     |
+    And the Tax Form dissapers
+    Then they see a Docusign expired session notification inside the iframe
+    And a refresh page button appears inside the iframe
+    And they must press the button to restart the Docusign process
+
+  @minutia @ui
+  Scenario: Docusign iframe throws an error
+    When the Docusign iframe throws one of the following error statuses:
+      | exception   |
+      | decline     |
+      | cancel      |
+      | fax_pending |
+    Then an error icon and message show in the iframe
+    And a "Refresh Page" button appears
+    And the participant must press the refresh button to continue
+
+  @minutia
+  Scenario: Participant decides to go back to step 2
```

Comment:
Is this still the case? I thought we removed this - the spec probably should be that they can't go back